### PR TITLE
feat:order details sold by section

### PR DESCRIPTION
**Pull request by alexj105**
The type of this PR is:Feature

This PR resolves [PX-4075]

### Description
Changed order status to fulfillment, get estimatedDelivery field, showed it to user
Add SoldBy section to Order details

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[PX-4075]: https://artsyproduct.atlassian.net/browse/PX-4075
ezgif com-gif-maker (1)

### CODE REVIEW

File: src/lib/Scenes/OrderHistory/OrderDetails/Components/SoldBySection.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface Props {
+  soldBy: SoldBySection_soldBy
+  testID?: string
+}
+
+export const SoldBySection: React.FC<Props> = ({ soldBy }) => {
+  console.log(soldBy, "soldBy")
```
Comment (kiryl-zubarau) [inline src/lib/Scenes/OrderHistory/OrderDetails/Components/SoldBySection.tsx:14]: #minor can you remove this?

File: src/lib/Scenes/OrderHistory/OrderDetails/Components/SoldBySection.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  const { fulfillments, artwork } = extractNodes(soldBy.lineItems)[0]
+  // const shippingOrigin =
```
Comment (kiryl-zubarau) [inline src/lib/Scenes/OrderHistory/OrderDetails/Components/SoldBySection.tsx:16]: #minor and this?

**Review by kiryl-zubarau — APPROVED** [review]
LGTM, just a few minor things.

File: src/lib/Scenes/OrderHistory/OrderDetails/__tests__/SoldBySection-tests.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      variables={{}}
+      render={({ props }) => {
+        if (props?.commerceOrder) {
+          return <SoldBySectionFragmentContainer soldBy={props.commerceOrder} />
+        }
+        return null
+      }}
+    />
+  )
+  it("renders auction result when auction results are available", () => {
```
Comment (brainbicycle) [inline src/lib/Scenes/OrderHistory/OrderDetails/__tests__/SoldBySection-tests.tsx:34]: copy paste issue? 
```suggestion
  it("renders sold by section and formats content", () => {
```

**Review by brainbicycle — APPROVED** [review]
Code looks good 👍, it looks like from the screenshots this view could use some bottom padding compared to the specs

File: src/lib/Scenes/OrderHistory/OrderDetails/__tests__/SoldBySection-tests.tsx (hunk 3 above)
Comment (alexj105) [reply src/lib/Scenes/OrderHistory/OrderDetails/__tests__/SoldBySection-tests.tsx:34]: > copy paste issue?

don't tell anybody :)

**Pull request merged**